gettingStarted.md

```diff
@@ -13,7 +13,7 @@ If you have deployed Pulsar cluster, you can get a list of machines for each com
 Before running Prometheus, you have to download a Prometheus image file and generate a config file.
 1. Download a Prometheus image at [Docker Hub](https://hub.docker.com/r/prom/prometheus), and install it.
 2. Generate a Prometheus config file. You can generate the config file with the following two options:
-- Copy the template file [prometheus/standalone.yml.template](prometheus/standalone.yml.template), and replace `{{ STANDALONE_HOST }}` with IP address of the machine running pulsar standalone.
+- Create the `prometheus.yml` file on your computer, copy the contents of the [template.standalone.yml](prometheus/standalone.yml.template) file to the `prometheus.yml` file, and replace `{{ STANDALONE_HOST }}` with IP address of the machine running pulsar standalone.
```

Comment:
```suggestion
- Create a `standalone.prometheus.yml` file on your computer, copy the contents of the [template.standalone.yml](prometheus/standalone.yml.template) file to the `standalone.prometheus.yml` file, and replace `{{ STANDALONE_HOST }}` with IP address of the machine running pulsar standalone.
```

Reply:
Or you can refine it as follows:
- Replicate the [template.standalone.yml](prometheus/standalone.yml.template) file on your computer, rename it as `standalone.prometheus.yml`, and replace `{{ STANDALONE_HOST }}` with IP address of the machine running pulsar standalone.

Reply:
Thanks, I've fixed.